test/test_1_ssl.py

```diff
@@ -13,11 +13,10 @@
         else:
             from urllib.request import urlopen
 
-        if sys.version_info[0:2] == (3, 3):
-            data = urlopen("https://www.nist.gov")
-        else:
-            context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
-            data = urlopen("https://www.nist.gov", context=context)
+        context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
```

Comment:
Why was this file renamed?

Reply:
To move on begin, to get error information faster (on `master` branch `test_ssl` is last executed)

Reply:
OK, right! I see it's reverted, now :-)